xml/http_servers.xml

```diff
@@ -4246,4 +4246,31 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:acme:mini_httpd:-"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^Unspecified, UPnP/[\d\.]+, Unspecified$">
+    <description> UPNP server</description>
+    <example>Unspecified, UPnP/1.0, Unspecified</example>
+    <param pos="0" name="service.family" value="UPnP"/>
+  </fingerprint>
+
+  <fingerprint pattern="^WNR2000v([0-9]) UPnP/[\d\.]+ miniupnpd/([\d\.]+)$">
+    <description>Netgear WNR2000v5 Router UPnP</description>
```

Comment:
Should `WNR2000` be `hw.product`?

Reply:
Yes, i would assume so. So i will add <param pos="0" name="hw.product" value="WNR2000"/> to it. Also should the hw_product.txt be updated with WNR2000?

Reply:
`hw_product.txt`? If you mean the description, then yes please.

Reply:
Doh, my brain was totally on the fields here. Thanks HD for the comment.

Reply:
recog_standardize identified some issue on another fingerprint in  http_server.xml for "Proxmox api daemon" they used multiple cases so it wants to create multiple Proxmox and proxmox in the identifiers. I will correct that fingerprint to use same case and rerun recog_standardize -w and push my correctlion up shortly.